parsers/casprrparser.py

```diff
@@ -0,0 +1,21 @@
+from utils.exceptions import InvalidFormat
```

Comment:
CASPRR format is identical to Psicov? If that's the case, we don't need to create a new parser, we just add the entry `CASPRR = PsicovParser` into `ContactFormats` and `ParserFormats`

Reply:
Remember to do this with any other contact formats that can be parsed with an already existing parser... We don't want to do the same work twice!